api/environments/urls.py

```diff
@@ -91,6 +92,12 @@
     WebhookConfigurationViewSet,
     basename="integrations-webhook",
 )
+
+environments_router.register(
+    r"edge-identities-with-identifier/featurestates",
```

Comment:
We could just have this as `"edge-identities/featurestates"`, right? Or `"edge-identities/edge-featurestates"` for consistency?

Reply:
We can't right, since we already have  viewsets mounted at those endpoints?

Reply:
I think we can? We have viewsets mounted at `/edge-identities/:uuid/edge-featurestates` but not at `/edge-identities/edge-featurestates`. I can't decide whether the consistency is good or whether it's just going to be confusing?